erpnext/patches/cc_change/add_allow_cost_center_in_entry_of_bs_account_field_in_account_settings.py

```diff
@@ -0,0 +1,26 @@
+# V 1.0.1
+
+from __future__ import unicode_literals
+import frappe
+from frappe.custom.doctype.custom_field.custom_field import create_custom_fields
+
+def execute():
+	make_custom_fields()
+
+def make_custom_fields():
+	allow_cost_center_in_entry_of_bs_account_field = [
```

Comment:
this should not be custom field, it should be a standard field in "Accounts Settings" (via .json)

Reply:
Yes, it should be thru json. I will remove patch to create custom field and add field via json.